src/main/scala/com/typesafe/sbt/site/Compat.scala

```diff
@@ -50,7 +50,7 @@ object Compat {
 
   def watchSettings(config: Configuration): Seq[Setting[_]] =
     Seq(
-      watchSources in Global += new Source(
+      watchSources += new Source(
```

Comment:
Hm. Why not `Global/watchSources`?

Reply:
`Global` doesn't exist anymore?

Reply:
I do not know much about `watchSources` but it looks like we need to use `ConfigGlobal` here:

https://github.com/sbt/sbt/blob/9e37391937b2d262dfb93325074dc692824519d1/main/src/main/scala/sbt/Defaults.scala#L405-L410